pkg/spec/gitops.go

```diff
@@ -675,8 +675,9 @@ func parseLabels(top map[string]json.RawMessage, result *GitOps, baseDir string,
 		if l.Name == "" {
 			multiError = multierror.Append(multiError, errors.New("name is required for each label"))
 		}
-		if l.Query == "" && len(l.Hosts) == 0 && l.HostVitalsCriteria == nil {
-			multiError = multierror.Append(multiError, errors.New("a SQL query, hosts list or host vitals criteria is required for each label"))
```

Comment:
I think we still want a check here -- if type is dynamic, it should have a query, and if it's host_vitals then it should have criteria.  We just won't need a check for `len(l.Hosts) == 0` anymore.